src/shared/ui/SectionList/SectionList.tsx

```diff
@@ -23,8 +23,8 @@ export const SectionList = ({ title, list, className }: SectionListProps) => {
             sm:text-[10.1px]/[15.15px]
             md:text-[15.325px]/[22.988px]
             xl:text-[24px]/[36px]">
-        {list.map((item) => (
-          <li key={item}>{item}</li>
+        {list.map((item, index) => (
```

Comment:
Great work, but in essence it's no different than using an item as an index, note that indexes are not always static. and check out this material https://www.developerway.com/posts/react-key-attribute.